src/category_theory/opposites.lean

```diff
@@ -229,6 +229,15 @@ protected definition op (α : F ≅ G) : G.op ≅ F.op :=
 @[simp] lemma op_hom (α : F ≅ G) : (nat_iso.op α).hom = nat_trans.op α.hom := rfl
 @[simp] lemma op_inv (α : F ≅ G) : (nat_iso.op α).inv = nat_trans.op α.inv := rfl
 
+protected definition unop (α : F.op ≅ G.op) : G ≅ F :=
```

Comment:
[The linter says this needs a docstring](https://github.com/leanprover-community/mathlib/pull/2132/checks?check_run_id=501759554#step:16:22): 
```
/- The `doc_blame` linter reports: -/
/- DEFINITIONS ARE MISSING DOCUMENTATION STRINGS: -/
-- category_theory/opposites.lean
#print category_theory.nat_iso.unop /- def missing doc string -/
```

Reply:
I've added some docs in 764fb73